common/autoware_motion_utils/src/trajectory_container/interpolator/akima_spline.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware/motion_utils/trajectory_container/interpolator/akima_spline.hpp"
+
+#include <Eigen/Dense>
+
+#include <cmath>
+#include <vector>
+
+namespace autoware::motion_utils::trajectory_container::interpolator
+{
+
+void AkimaSpline::compute_parameters(
+  const Eigen::Ref<const Eigen::VectorXd> & axis, const Eigen::Ref<const Eigen::VectorXd> & values)
+{
+  int32_t n = static_cast<int32_t>(axis.size());
```

Comment:
@yhisaki 

Can you add `const` to variables whose values will not change after initialization?
cf. [TIER IV INTERNAL LINK](https://tier4.atlassian.net/wiki/spaces/CT/pages/3042836632/P+C#Major)